tests/test_core.py

```diff
@@ -1940,9 +1938,11 @@ def dealloc(self):
         self.assertEqual(attr1.retainCount(), 1)
 
         # ObjC object will be deallocated, can only access Python attributes afterwards.
```

Comment:
Comment should be reworded or removed.

Reply:
Replaced now with a comment to explain why we need the autorelease pool.